server/model/order.js

```diff
@@ -0,0 +1,33 @@
+const utils = require('../utils/utils');
+
+const ordersList = [];
+
+const getAllOrders = () => (ordersList);
+
+const getOrder = (id) => {
+  if (ordersList.some((e) => e.id === id)) {
+    return ordersList.find((order) => order.id === id);
+  }
+  return false;
```

Comment:
returning two different types of values in a function is not a good practice,
if you want falsy value as we talked about you can use null, as null is typeof object

Reply:
you can just do this for this function
const getOrder = (id) => {
  return ordersList.find((order) => order.id === id);
}
as find function will return undefined when there is no item found, and undefined is falsy value